omc3/model/accelerators/petra.py

```diff
@@ -0,0 +1,29 @@
+"""
+PETRA
+-------------------
+"""
+import os
+import datetime as dt
```

Comment:
Never used, remove

Reply:
as discussed, model creation is the next step.